srcpkgs/nodejs/template

```diff
@@ -50,6 +50,7 @@ esac
 
 CFLAGS="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
 CXXFLAGS="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
+COV_SKIP_TESTS="core_line_numbers.js,testFinalizer.js,test_function/test.js,test-cluster-primary-error.js,test-cluster-primary-kill.js"
```

Comment:
Keep custom variables in a local function scope if possible (xbps-src doesn't always clean them up properly otherwise)

Reply:
outdated

Reply:
And does not work.